apps/www/_blog/2020-06-01-supabase-alpha-may-2020.mdx

```diff
@@ -14,15 +14,17 @@ video: https://www.youtube.com/v/e4qXmcEFaUs
 
 It has been a monster month at [Supabase](/) and we're back with a video update. It's a longer video this month because we have a couple of housekeeping items.
 
-<iframe
-  className="w-full"
-  width="700"
-  height="350"
-  src="https://www.youtube-nocookie.com/embed/e4qXmcEFaUs"
-  frameBorder="0"
-  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-  allowfullscreen
-></iframe>
+<div className="video-container">
+  <iframe
+    className="w-full"
+    width="700"
```

Comment:
I think we can remove width and height from everything now?

Reply:
Just removed